src/main/kotlin/lotto/common/MoneyCalculator.kt

```diff
@@ -0,0 +1,8 @@
+package lotto.common
+
+object MoneyCalculator {
+
+    fun calculateRateOfReturn(purchasedMoney: Long, earnedMoney: Long): Double {
+        return earnedMoney.toDouble() / purchasedMoney.toDouble() * 100.0
+    }
```

Comment:
What do you prefer both of expression?

```suggestion
    fun calculateRateOfReturn(purchasedMoney: Long, earnedMoney: Long)  = earnedMoney.toDouble() / purchasedMoney.toDouble() * 100.0
   
```

Reply:
Since it's a single line and easy logic, I think just using `=` could be more readable.